README.md

```diff
@@ -177,8 +177,7 @@ from the [`openverse_catalog`][cc_airflow] directory.
 To reset the test DB (wiping out all databases, schemata, and tables), run
 
 ```shell
-docker-compose down
-rm -r /tmp/docker_postgres_data/
+docker-compose down -v
```

Comment:
Might also be good to mention the [`docker volume prune`](https://docs.docker.com/engine/reference/commandline/volume_prune/) command here which might be useful when Docker containers are already stopped.

Reply:
Oh good call!